### PR TITLE
fix vfmt

### PR DESCRIPTION
**Pull request by donnisnoni95**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "semi": true,
   "singleQuote": true,
-  "tabWidth": 2,
+  "tabWidth": 4,
```
Comment (Alhadis) [inline .prettierrc]: Tabs should ideally be used too, as per V's styleguide.

Comment (donnisnoni95) [reply .prettierrc]: agree with you @Alhadis

Comment (Alhadis) [reply .prettierrc]: Tabs, because they're better than spaces. 😉

Scientifically-proven fact.

**Comment by Alhadis** [comment]
You'll probably also want to rerun Prettier with the updated config. 😉

File: src/client/extension.ts [hunk 2]
```diff
@@ -15,86 +13,13 @@ export function activate(context: vscode.ExtensionContext) {
...
-  let testPackage = vscode.commands.registerCommand(
-    'v.test.package',
-    commands.testPackage
-  );
-  let playground = vscode.commands.registerCommand(
-    'v.playground',
-    commands.playground
-  );
-
+  let testFile = vscode.commands.registerCommand('v.test.file', commands.testFile);
```
Comment (alexesprit) [inline src/client/extension.ts]: Also, you could get rid of result values, as they aren't used actually:
```ts
vscode.commands.registerCommand('v.test.file', commands.testFile);
```

Comment (donnisnoni95) [reply src/client/extension.ts]: @alexesprit Yeah... I think so...

**Comment by donnisnoni95** [comment]
@0x9ef Can we release now ??

**Comment by 0x9ef** [comment]
Oh, i fucked up with my keyboard, sorry :(

**Pull request merged**